.github/workflows/analyze.yml

```diff
@@ -8,16 +8,11 @@ on:
 
 jobs:
```

Comment:
Can be a follow-up, but maybe we should just use the same workflow as for most of the other repos, to be consistent? (and just remove the steps we don't need) I.e. https://github.com/ubuntu/app-center/blob/main/.github/workflows/ci.yaml